kafka_utils/kafka_check/commands/offline.py

```diff
@@ -62,6 +64,10 @@ def _prepare_output(partitions, verbose):
                 for (topic, partition) in partitions
             )
             out['verbose'] = "Partitions:\n" + "\n".join(lines)
+        else:
+            cmdline = sys.argv[:]
+            cmdline.insert(1, '-v')
+            out['message'] += '\nTo see all offline partitions run: ' + ' '.join(cmdline)
```

Comment:
Perhaps we could simply print "use --verbose (-v) to see all offline partitions"